packages/vc-http-api-test-server/__tests__/issueCredential.spec.js

```diff
@@ -69,15 +69,15 @@ if (suiteConfig.issueCredentialConfiguration) {
             expect(res.status).toBe(400);
         });
 
-        it(`6. The Issuer's Issue Credential HTTP API MUST reject if the value of "options.assertionMethod" in the body of the POST request does not exist.`, async () => {
+        it(`6. The Issuer's Issue Credential HTTP API MUST reject if the value of "options.verificationMethod" in the body of the POST request does not exist.`, async () => {
             const body = {
               credential: {
                 ...credentials[0].data,
                 issuer: value.id
               },
               options: {
                 ...value.options[0],
-                assertionMethod: 'foo',
+                verificationMethod: 'foo',
```

Comment:
I'm a bit concerned here... at present, the only verification relationship that can issue a VC is "assertionMethod". Why are we generalizing here? What's the use case?

Reply:
Because the test currently does not match the spec: https://github.com/w3c-ccg/vc-http-api/blob/master/docs/vc-http-api.yml#L228

Alternative solution would be to update the spec.

Reply:
Yes, this PR is just to fix an inconsistency with the current spec.

Reply:
Right, sorry -- looks like the test is wrong and needs to be fixed. Agree with @OR13 that it's a bit late to do that.

I'm confused as to why DB is passing this test as-is... we should be failing it, no? /cc @gannan08?